server/catgenome/src/main/resources/database/catgenome/postgres/v2021.11.03_12.00__lineage_tree.sql

```diff
@@ -0,0 +1,32 @@
+CREATE SEQUENCE IF NOT EXISTS CATGENOME.S_LINEAGE_TREE START WITH 1 INCREMENT BY 1;
```

Comment:
Please, run some tests on Postgres to make sure that migrations and queries work fine for this database as well.